### PR TITLE
Connect monitoring app to table data base

### PR DESCRIPTION
**Pull request by smartsena**

### CODE REVIEW

File: cmr2_api/settings.py [hunk 1]
```diff
@@ -96,18 +96,19 @@
         'HOST': os.getenv('DB_HOST'),
         'PORT': os.getenv('DB_PORT', 5432)
     },
-    'priority_database': {
+    'database_for_reading': {
```
Comment (dagnaldo) [inline cmr2_api/settings.py:99]: `bd_for_read`

File: docker-compose.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ services:
       POSTGRES_DB: testdb
       POSTGRES_PASSWORD: testpassword
   
-  db_priority:
+  db_FOR_READING:
```
Comment (dagnaldo) [inline docker-compose.yml:11]: `db_for_read`

Comment (dagnaldo) [reply docker-compose.yml:11]: add on `depends_on`, maybe...

**Review by dagnaldo — CHANGES_REQUESTED** [review]
Take a look on requests.

File: monitoring/models.py [hunk 3]
```diff
@@ -63,13 +57,55 @@ class MonitoringConsolidated(models.Model):
...
+
+    nu_longitude = models.DecimalField(
+        _('Longitude'),
+        max_digits=20,
+        decimal_places=15,
+        null=True,
+        blank=True,
+    )
+
+    # co_cr = models.ForeignKey(
```
Comment (dagnaldo) [inline monitoring/models.py:97]: please, remove comments.

File: monitoring/models.py [hunk 4]
```diff
@@ -79,13 +115,18 @@ class MonitoringConsolidated(models.Model):
         blank=True,
     )
 
-    co_funai = models.ForeignKey(
-        LimiteTerraIndigena,
+    # co_funai = models.ForeignKey(
```
Comment (dagnaldo) [inline monitoring/models.py:118]: same here.

File: monitoring/serializers.py [hunk 5]
```diff
@@ -15,11 +15,15 @@ class Meta:
         id_field = False
         fields = (
             'id',
-            'tb_ciclo_monitoramento_id',
             'no_imagem',
             'dt_imagem',
             'no_estagio',
-            'dt_cadastro',
+            'no_ciclo',
```
Comment (dagnaldo) [inline monitoring/serializers.py:21]: Do you guys, really need all fields on properties?

The behave expected when you click on geometry is to call an api request, right? 
So... I think that you dont need all fields here. Less is more!

File: monitoring/serializers.py [hunk 6]
```diff
@@ -40,11 +44,15 @@ class Meta:
...
```
Comment (dagnaldo) [inline monitoring/serializers.py:50]: Here is fine, because of the detailed request.

File: monitoring/serializers.py [hunk 7]
```diff
@@ -83,11 +91,15 @@ class Meta:
...
```
Comment (dagnaldo) [inline monitoring/serializers.py:97]: The table request need all fields?

**Review by dagnaldo — CHANGES_REQUESTED** [review]
I need to see some new changes again.

When you finish the request, please paste the commit code on each conversation and resolve it.

File: monitoring/models.py [hunk 8]
```diff
@@ -1,4 +1,4 @@
-from funai.models import CoordenacaoRegional, LimiteTerraIndigena
+# from funai.models import CoordenacaoRegional, LimiteTerraIndigena
```
Comment (dagnaldo) [inline monitoring/models.py:1]: remove unecessary imports and comments.

File: cmr2_api/settings.py (hunk 1 above)
Comment (smartsena) [reply cmr2_api/settings.py]: OK

File: docker-compose.yml (hunk 2 above)
Comment (smartsena) [reply docker-compose.yml]: OK

File: monitoring/serializers.py (hunk 7 above)
Comment (smartsena) [reply monitoring/serializers.py:97]: OK

File: monitoring/serializers.py (hunk 6 above)
Comment (smartsena) [reply monitoring/serializers.py:50]: OK

File: monitoring/serializers.py (hunk 5 above)
Comment (smartsena) [reply monitoring/serializers.py:21]: OK

File: monitoring/models.py (hunk 3 above)
Comment (smartsena) [reply monitoring/models.py:97]: OK

File: monitoring/models.py (hunk 4 above)
Comment (smartsena) [reply monitoring/models.py:118]: OK

File: monitoring/models.py (hunk 8 above)
Comment (smartsena) [reply monitoring/models.py:1]: OK

**Review by dagnaldo — APPROVED** [review]

**Pull request merged**